src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -86,13 +86,11 @@
 
         // intake can ids are range 30-39
         public static final int INTAKE_MOTOR = 30, INTAKE_MOTOR_2 = 31, INTAKE_SOLENOID_UP = 1,
-                INTAKE_SOLENOID_DOWN = 0;
+                INTAKE_SOLENOID_DOWN = 0, INTAKE_PROXIMITY = 8;
```

Comment:
Might have been good to leave a reminder that the proximity sensor IDs are placeholder values, but it's probably fine.